vagrant-serverspec.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |gem|
   #add onlly dependencies for winrm without nokogiri
   gem.add_runtime_dependency 'winrm', '~> 2.1', '>= 2.1'
   gem.add_runtime_dependency 'os', '~> 0.9.6'
+  gem.add_runtime_dependency 'rspec_html_formatter', '~> 0.3', '>= 0.3.1'
```

Comment:
OK it sounds great.
You can find here
[rspec_html_reports.zip](https://github.com/vvchik/vagrant-serverspec/files/1054931/rspec_html_reports.zip) a zip file that contains an "overview.html" file summary of serverspec test plan results executed by vagrant-serverspec. You can extract all files into a directory and browse this file.
It's very usefull to integrate that report with jenkins job as I use it myself.